pkg/cmd/provisioning/utils.go

```diff
@@ -23,3 +27,34 @@ func EnsureDir(path string) error {
 
 	return nil
 }
+
+// CreateShellScript creates a shell script given commands to execute
+func CreateShellScript(commands []string) string {
+	return fmt.Sprintf("#!/bin/sh\n%s", strings.Join(commands, "\n"))
+}
+
+// SaveToFile saves the given data to a given file
+func SaveToFile(filePurpose, filePath string, data []byte) error {
```

Comment:
Any particular reason to not simply use ioutil.WriteFIle()?

Reply:
to be honest I just moved it to utils.go since it was a common code. Let me explore if we can use ioutil.WriteFile() here.

Reply:
done